hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/contracts/operations/HederaEvmOperationsUtil.java

```diff
@@ -58,6 +67,9 @@ static Operation.OperationResult addressCheckExecution(
         } catch (final FixedStack.UnderflowException ufe) {
             return new Operation.OperationResult(
                     supplierHaltGasCost.getAsLong(), ExceptionalHaltReason.INSUFFICIENT_STACK_ITEMS);
+        } catch (final FixedStack.OverflowException ofe) {
+            return new Operation.OperationResult(
```

Comment:
This to me indicates we are not checking for overflow in one of the system account execution providers.  We should address it where it happens instead of letting it throw across stack frames, it will perform better.